### PR TITLE
[#155535535] project cleanup

### PR DESCRIPTION
**Pull request by egichuri**
https://www.pivotaltracker.com/story/show/155535535

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[flake8]
+exclude = .venv,migrations
+max-complexity = 5
```
Comment (wanjiku-carol) [inline .flake8]: Missing line at the end of file

Comment (egichuri) [reply .flake8]: oops

**Comment by wanjiku-carol** [comment]
LGTM

**Comment by nelmandela** [comment]
LGTM

File: settings/base.py [hunk 2]
```diff
@@ -32,14 +32,10 @@
...
-SECRET_KEY = 'SECRET_KEY'
 
 SECRET_KEY = config('SECRET_KEY')
-DATABASES = {
-    'default': dj_database_url.config(
-        default=config('DATABASE_URL')
-    )
-}
+
+DATABASES = {'default': dj_database_url.config()}
```
Comment (HawiCaesar) [inline settings/base.py]: Will this allow flexibility to allow each dev to use their own environment variables ?

Comment (wanjiku-carol) [reply settings/base.py]: LOL! I've merged! But yes @HawiCaesar it should.

Comment (egichuri) [reply settings/base.py]: @HawiCaesar should be fine as long as you have `DATABASE_URL` env variable defined

**Pull request merged**